packages/requirements-optional.txt

```diff
@@ -1,5 +1,5 @@
-torch_geometric==2.3.0
--f https://data.pyg.org/whl/torch-2.2.0+cpu.html
+torch_geometric==2.5.3
+-f https://data.pyg.org/whl/torch-2.4.0+cpu.html
```

Comment:
is this meant to be 2.4.1 or 2.4.0?

Reply:
2.4.0 , following instructions from https://pytorch-geometric.readthedocs.io/en/latest/notes/installation.html